linting/eslint.config.js

```diff
@@ -2,7 +2,6 @@
  * Copyright Strimzi authors.
```

Comment:
Note the file path move; @dlabaj PR had updated in original linting location, moved it over to our main one now